docs/DeveloperGuide.md

```diff
@@ -2,33 +2,73 @@
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}\
+This section describes some noteworthy details on how certain features are implemented.
 
 
 ## Product scope
 ### Target user profile
 
 {Describe the target user profile}
+* NUS students (especially catering to Freshman)
+* has a need to manage their school related tasks, classes and notes
+* prefer desktop apps over other types
+* can type fast
+* prefers typing to mouse interactions
+* is reasonably comfortable using CLI apps
 
 ### Value proposition
 
 {Describe the value proposition: what problem does it solve?}
+* This app simplifies the plethora of information that freshman are bombarded with for easy viewing.
```

Comment:
Perhaps can add "streamlines information for easy viewing and management"?

Reply:
Thanks, added